### ISSUE TITLE
Code Review Tutorial Series

### ISSUE THREAD

**Issue by krixano** [krixano/ZeroNet-Dev-Center#20]
It might be cool to do some type of tutorial series that goes through an existing zite's code step by step and explaining everything that it does. It may help people understand how current zites do things, especially database-wise, etc. Just an interesting though I have. Will most definitely be a long series and take a lot of work to create though.

**Comment by imachug** [comment]
Any ideas about what site to take? AFAIK ZeroSites is a good choice.

**Comment by krixano** [comment]
One of my zites would probably be the easiest. However, I use VueJs, Gulp, Browserify, etc. which a lot of people might not know. I think if something like this were to be written, it would be better to have something more advanced than ZeroSites, imo. I feel like ZeroSites wouldn't benefit from this style because it's too simple. I think this style of tutorial would be better for bigger zites (Git Center, ZeroMedium, ZeroMe, etc.)

**Comment by imachug** [comment]
> I think this style of tutorial would be better for bigger zites (Git Center, ZeroMedium, ZeroMe, etc.)
I agree here, but you said `goes through an existing zite's code step by step and explaining everything that it does`, so you have to explain each line of `ZeroMe`/`ZeroMedium`/`Git Center` and there are many lines.

**Comment by AnthyG** [comment]
I found ZeroMe's code is quite interesting for DB-Stuff in general, and also for the Merger-functionality.
ZeroMail is especially good to get used to the different ways of encryption, ZeroNet provides in-house.
And just to note it, both zites original code is in coffee-script, which I didn't find that hard to read at all (though I would probably never use it for myself :laughing:)!

**Comment by imachug** [comment]
From that point of view `ZeroMe` is the best choice.
